Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -147,6 +189,10 @@ extension HTTPClient {
         public let scheme: String
         /// Remote host, resolved from `URL`.
         public let host: String
+        /// Socket path, resolved from `URL`.
+        public let socketPath: String
+        /// URI, resolved from `URL`.
+        public let uri: String
```

Comment:
Is it necessary for either of these to be `public`? In particular, `uri` is _extremely_ not well distinguished from `url`, and if we're keeping both in the API we really need to explain how they differ.

Reply:
Not at all — continuing on that thought, the host will be empty for socket-based Requests, even though that's where its encoded… Should we merge the implementations to make the socket path available as the `host`?